src/acl.c

```diff
@@ -220,6 +220,12 @@ uint64_t ACLGetCommandCategoryFlagByName(const char *name) {
     return 0; /* No match. */
 }
 
+/* Method for passwords/pattern comparison used for the user->passwords list
+ * so that we can search for items with listSearchKey(). */
+int ACLListMatchUser(void *a, void *b) {
+    return sdscmp(((sds *) a)[0],((sds *) b)[0]) == 0;
```

Comment:
this smells a bit. for a moment i thought there's some struct in which the first member happens to be an sds.
how about:
```suggestion
int ACLListMatchUser(void *a, void *b) {
    sds *user_rules = a;
    sds name = b;
    return sdscmp(user_rules[0], name) == 0;
```
and then do:
```diff
-    if (listSearchKey(UsersToLoad, argv+1)) {
+   if (listSearchKey(UsersToLoad, argv[1])) {
```

Reply:
Sorry, I sometimes do git push out of a habit when I'm taking a break, I simplified it a bit and also documented the expectation a bit more.